### ISSUE TITLE
`check_env` fails due to missing random number generator

### ISSUE THREAD

**Issue by adzcai** [hsahovic/poke-env#333]
I'm following the tutorial on [getting started with RL and `EnvPlayer`s](https://poke-env.readthedocs.io/en/latest/rl_with_open_ai_gym_wrapper.html), but running into the following error when I call `check_env(test_env)`:

```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
/Users/alexandercai/Developer/game/ps-rl/PokemonShowdown.ipynb Cell 3 in <cell line: 3>()
      [1](vscode-notebook-cell:/Users/alexandercai/Developer/game/ps-rl/PokemonShowdown.ipynb#W2sZmlsZQ%3D%3D?line=0) opponent = RandomPlayer(battle_format='gen8randombattle')
      [2](vscode-notebook-cell:/Users/alexandercai/Developer/game/ps-rl/PokemonShowdown.ipynb#W2sZmlsZQ%3D%3D?line=1) test_env = SimpleRLPlayer(opponent=opponent, battle_format='gen8randombattle', start_challenging=True)
----> [3](vscode-notebook-cell:/Users/alexandercai/Developer/game/ps-rl/PokemonShowdown.ipynb#W2sZmlsZQ%3D%3D?line=2) check_env(test_env)

File /usr/local/Caskroom/miniconda/base/envs/dev/lib/python3.9/site-packages/gym/utils/env_checker.py:282, in check_env(env, warn, skip_render_check)
    279 check_space_limit(env.observation_space, "observation")
    281 # ==== Check the reset method ====
--> 282 check_reset_seed(env)
    283 check_reset_options(env)
    284 check_reset_info(env)

File /usr/local/Caskroom/miniconda/base/envs/dev/lib/python3.9/site-packages/gym/utils/env_checker.py:80, in check_reset_seed(env)
     76 obs_1 = env.reset(seed=123)
     77 assert (
     78     obs_1 in env.observation_space
     79 ), "The observation returned by `env.reset(seed=123)` is not within the observation space."
---> 80 assert (
     81     env.unwrapped._np_random  # pyright: ignore [reportPrivateUsage]
     82     is not None
     83 ), "Expects the random number generator to have been generated given a seed was passed to reset. Mostly likely the environment reset function does not call `super().reset(seed=seed)`."
     84 seed_123_rng = deepcopy(
     85     env.unwrapped._np_random  # pyright: ignore [reportPrivateUsage]
     86 )
     88 obs_2 = env.reset(seed=123)

AssertionError: Expects the random number generator to have been generated given a seed was passed to reset. Mostly likely the environment reset function does not call `super().reset(seed=seed)`.
```

My code is identical to what's provided in the tutorial. I noticed that the `OpenAIGymEnv` overrides the native `Env.seed` method, which isn't setting `self._np_random` in the environment. Is there any reason to override that method? Seems that using the super `Env.seed` would fix the issue.

**Comment by cameronangliss** [comment]
@hsahovic this is fixed now because of #467 right?

**Comment by hsahovic** [comment]
Good catch, thanks @cameronangliss !
